keras_cv/backend/__init__.py

```diff
@@ -52,7 +52,13 @@
 
     keras.backend.name_scope = keras.name_scope
 else:
-    from tensorflow import keras
+    try:
```

Comment:
We may want to be more conservative to enable people to use Keras 2 via tf.keras without having to install tf_keras:

1. `from tensorflow import keras`
2. `if not hasattr(keras, "version"):` then `tf.keras` is Keras 2. We use `keras` from TF.
3. else: try to import `tf_keras`
4. if not available: raise